app/assets/stylesheets/styles/new_game.scss

```diff
@@ -1,3 +1,16 @@
 #newform {
-  padding: 10px;
+  padding: 50px;
+}
+
+.form-inputs label {
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.